### PR TITLE
Feature/frontend register

### PR DESCRIPTION
**Pull request by Harshtroo**

### CODE REVIEW

File: tms/static/js/register.js [hunk 1]
```diff
@@ -5,20 +5,25 @@ $("#user_register").on("submit",function(event){
...
-
         setTimeout(function () {
             messageElement.fadeOut(500, function () {
                 $(this).remove();
             });
         }, 5000);
     };
      var callback = function(response) {
+                        debugger
                         if (response.status === "success") {
```
Comment (Dhruvi-Makwana) [inline tms/static/js/register.js]: remove debugger

File: tms/static/js/register.js [hunk 2]
```diff
@@ -6,12 +6,25 @@ $("#user_register").on("submit",function(event){
...
+        }, 5000);
+    };
      var callback = function(response) {
-
-                         window.location.href = baseURL + "/login/";
+                        if (response.status === "success") {
+                            showMessage("Registration successful!", "green");
+                            window.location.href = baseURL + "/login/";
+                        } else if (response.status === "error") {
+                            showMessage("Error: " + response.message, "red");
```
Comment (Dhruvi-Makwana) [inline tms/static/js/register.js]: why triple ==== you can user ==

File: tms/static/js/base.js [hunk 3]
```diff
@@ -0,0 +1,14 @@
+var baseURL = "http://127.0.0.1:8000"
```
Comment (yatishTrootech) [inline tms/static/js/base.js]: it should not be full url

File: tms/static/js/register.js [hunk 4]
```diff
@@ -0,0 +1,35 @@
+
+var registerURL = baseURL + '/register/'
```
Comment (yatishTrootech) [inline tms/static/js/register.js]: no need of base url

File: tms/static/js/register.js [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+                        "last_name" :$("#last_name").val(),
+                        "username" :$("#username").val(),
+                        "email":$("#email").val(),
+                        "groups":$("#groups").val(),
+                        "password":$("#password").val(),
+                        "password_confirm":$("#password_confirm").val()}
+
+     var showMessage = function (message, color) {
+        var messageElement = $("<div>").text(message).css("color", color);
+        $("#message-container").html(messageElement);
```
Comment (yatishTrootech) [inline tms/static/js/register.js]: it should be in base

File: tms/tms/settings/base.py [hunk 6]
```diff
@@ -162,6 +162,7 @@
     "clickup",
     "rest_framework",
     "rest_framework.authtoken",
+    "django_summernote",
```
Comment (yatishTrootech) [inline tms/tms/settings/base.py:165]: add ss/videos from of you functionality

**Review by yatishTrootech — CHANGES_REQUESTED** [review]

**Comment by Harshtroo** [comment]
[register.webm](https://github.com/Harshtroo/tms/assets/123172610/4da1740a-afd3-4984-bd74-1737d2d18241)

**Review by yatishTT — APPROVED** [review]

**Pull request merged**